modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -140,6 +154,20 @@ public Connection createConnection(String queryString) throws SQLException {
         }
     }
 
+    /**
+     * Template method for constructing the JDBC URL to be used for creating {@link Connection}s.
+     * This should be overridden if the JDBC URL and query string concatenation or URL string
+     * construction needs to be different to normal.
+     *
+     * @param queryString
+     *          query string parameters that should be appended to the JDBC connection URL.
+     *          The '?' character must be included
+     * @return a full JDBC URL including queryString
+     */
+    protected String constructUrlForConnection(String queryString) {
```

Comment:
This change is unrelated, isn't it?

Reply:
No, and I haven't done them. Will try to revert...